src/io/content-type.js

```diff
@@ -0,0 +1,34 @@
+import mime from 'mime-types';
```

Comment:
This [file](https://github.com/humphd/next/blob/0701d854f8118e99ea7ba1c33ac0329cc8f38cf1/src/web-server/content-type.js#L27) already exists on the web-server. Please reuse it.

Reply:
@dsych Not sure how to do that, care to help?